addons/website_sale/models/website.py

```diff
@@ -219,7 +219,9 @@ def get_pricelist_available(self, show_visible=False):
         is_user_public = self.env.user._is_public()
         if not is_user_public:
             last_order_pricelist = partner_sudo.last_website_so_id.pricelist_id
-            partner_pricelist = partner_sudo.property_product_pricelist
+            partner_pricelist = partner_sudo.with_context(
+                geoip_country=country_code,
+            ).property_product_pricelist
```

Comment:
Nitpicking in case geoip isn't enabled/working, only update the context if `country_code` has a truthy value?
Since `depends_context` splits the cache according to the context value, I want to make sure the field is not recomputed twice because the key is unset first, and then it's set to whatever falsy value geoip can return.